LEGO1/mxatomid.h

```diff
@@ -1,8 +1,14 @@
 #ifndef MXATOMID_H
 #define MXATOMID_H
 
+#include "mxtypes.h"
+
 enum LookupMode
 {
+    Exact = 0,
+    LowerCase = 1,
+    UpperCase = 2,
+    LowerCase2 = 3
 };
```

Comment:
MSVC4.2 does not supoprt e.g. `LookupMode::Exact`.
So perhaps I should namespace its member values even further.
e.g. `LookupMode::LookupMode_Exact`